modules/users/server/config/strategies/local.js

```diff
@@ -22,7 +22,7 @@ module.exports = function () {
       }
       if (!user || !user.authenticate(password)) {
         return done(null, false, {
-          message: 'Invalid username or password'
+          message: 'Invalid username or password (' + (new Date()).toLocaleTimeString() + ')'
```

Comment:
I'd rather the client application be given the responsibility to provide that UI feedback. Most likely it should be in the form of an animation/ui-effect/sound, or something similar. Users of this framework will probably have different ways of conveying that the page did indeed reload or attempt to login again.

We should remove the date/time stamp from this piece.